src/test/CMakeLists.txt

```diff
@@ -104,8 +112,7 @@ target_link_libraries(ceph_bench_log global pthread rt ${BLKID_LIBRARIES} ${CMAK
 add_executable(ceph_test_mutate
   test_mutate.cc
   )
-target_link_libraries(ceph_test_mutate global librados ${BLKID_LIBRARIES}
-  ${CMAKE_DL_LIBS})
+target_link_libraries(ceph_test_mutate global librados ${BLKID_LIBRARIES} ${CMAKE_DL_LIBS})
```

Comment:
@tchaikov 
Probably a leftover from this PR, and perhaps someboy has wrapped this line.
OR I tried to make to make it consistent with the lines around it.

I do have a some question marks about long line wrapping, but addressing that right now is not really in my focus